src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -963,6 +964,10 @@ void grpc_chttp2_mark_stream_writable(grpc_chttp2_transport* t,
 static grpc_closure_scheduler* write_scheduler(grpc_chttp2_transport* t,
                                                bool early_results_scheduled,
                                                bool partial_write) {
+  // If we're already in an event manager, don't offload this to an executor
```

Comment:
Not sure whether you want to use the word "event manager" here. What I usually use is "background poller".

Otherwise the changes here look good to me.

Reply:
Changed. Thanks for the comment!